indexer/versionedscanner.go

```diff
@@ -110,8 +110,28 @@ func (vs VersionedScanners) VStoRS() []RepositoryScanner {
 	return out
 }
 
+func (vs *VersionedScanners) FStoVS(scnrs []FileScanner) {
```

Comment:
is this meant to overwrite the previous contents?

Reply:
It is, probably worth looking at this and how it's used in the controller.Coalescer again in a different PR, adding missing docs.